src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -53,6 +50,12 @@ object ModuleNames {
   final val NAMED_SERVER_SET_PATH = "SERVER_SET_PATH"
   final val NAMED_SERIALIZE_GROUP_UPDATES = "SERIALIZE_GROUP_UPDATES"
   final val NAMED_HTTP_EVENT_STREAM = "HTTP_EVENT_STREAM"
+
+  final val STORE_APP = "AppStore"
```

Comment:
the other constants are prefixed with "NAMED_". It would be nice to adhere to that convention.